api/src/main/resources/liquibase.xml

```diff
@@ -167,4 +167,16 @@
 			ON (A.encounter_id = B.encounter_id));
 		</sql>
 	</changeSet>
+	<changeSet id="ugandaemr-17102017-0839" author="slubwama">
+		<preConditions onFail="CONTINUE">
+			<sqlCheck expectedResult="1">
+				SELECT COUNT(*) FROM concept WHERE concept_id = '164985'
+			</sqlCheck>
+		</preConditions>
+		<comment>Migrate HTC Identifier value to Concept</comment>
```

Comment:
@slubwama this query copies both the HTC and OpenMRS ID - so the subquery needs to only return the value of the HTC identifier not all identifiers for the patient